### PR TITLE
Fix the cfg variable configuration bug.

### PR DESCRIPTION
**Pull request by pettershao-ragilenetworks**
#### Why I did it
var cfg was declared in line 62. Otherwise, the configuration will be abnormal.
#### How I did it
var cfg was declared in line 62. Otherwise, the configuration will be abnormal.
#### How to verify it
var cfg was declared in line 62. Otherwise, the configuration will be abnormal.
#### Which release branch to backport (provide reason below if selected)

- [ ] 201811
- [ ] 201911
- [ ] 202006
- [ ] 202012
- [ ] 202106
- [ ] 202111

### CODE REVIEW

**Comment by pettershao-ragilenetworks** [comment]
@ganglyu & @zbud-msft - Please provide your review/approval.

**Comment by ganglyu** [comment]
Can you explain the configuration will be abnormal? What's the behavior?

**Comment by pettershao-ragilenetworks** [comment]
ca0e5a3d-d572-499e-be59-13b74ec6df39
2cd9ce67-6d09-4051-8002-5ff206014f82
398cb523-1c8b-4863-a72c-3efa20c47733

pls see log, it is two different variables with the same name.

**Comment by pettershao-ragilenetworks** [comment]
@ganglyu

File: telemetry/telemetry.go [hunk 1]
```diff
@@ -135,7 +135,6 @@ func main() {
 	}
 
 	opts = []grpc.ServerOption{grpc.Creds(credentials.NewTLS(tlsCfg))}
-	cfg := &gnmi.Config{}
 	cfg.Port = int64(*port)
 	cfg.UserAuth = userAuth
```
Comment (ganglyu) [inline telemetry/telemetry.go:139]: We only need UserAuth here, can you remove line 138, 140 and 141?

**Review by ganglyu — APPROVED** [review]

**Pull request merged**